src/Common/Internal/Http/HttpClient.php

```diff
@@ -84,6 +84,8 @@ public function __construct(
             Resources::USE_BRACKETS => true,
             Resources::SSL_VERIFY_PEER => false,
             Resources::SSL_VERIFY_HOST => false,
+            'proxy_host' => "localhost",
```

Comment:
I think having something like HTTP_PROXY=http://localhost:8888 makes sense for this feature. Instead of commenting out the code, can you look for environment variable HTTP_PROXY? if it exists, parse it out and use its host and port values.